### ISSUE TITLE
Redirect to different page after login

### ISSUE THREAD

**Issue by MarkRobertJohnson** [frankchen76/EZCode-Adal-Angular5#4]
Is there a way to redirect to a different page after login?  Currently it appears to only be able to redirect to '' due to the fact the ADAL includes a hash character in the reply before the id_token=.   I can put in something like "http://localhost:4200/admin" for the reply URL, but because it has no hash (i.e. #/admin), it will just redirect to ''.

**Comment by frankchen76** [comment]
you can specify a redirectUri in config. AAD will redirect to that page where all your SPA logic will be in that page. the router.navigate(['']) just point to the route root which you defined in the route. 

```
export const ezcodeAdalConfigLocal: IEZCodeAdalConfig={
    tenant: '[tenantid]',
    clientId: '[clientId]',
    redirectUri: 'http://localhost:4200/admin', //window.location.href.substring(0, window.location.href.lastIndexOf("/")+1)
    postLogoutRedirectUri: window.location.origin + '/',
    endpoints: {
		...
    }
};
```
let me know if that works.

**Comment by MarkRobertJohnson** [comment]
I was trying "'http://localhost:4200/admin'" but it has to be "'http://localhost:4200/#/admin'" due to the requirement of using HashLocationStrategy, unless that is not a requirement?

**Comment by frankchen76** [comment]
Yes, there is a requirement to use HashLocationStrategy. Were you trying to change the redirection page or redirection route? if you were asking the page redirect, you can change the redirectUri to different page. this requires you to host your SPA on that page when you run `ng serve`. If you were asking the route redirect, you can add the following code to redirect default '' route to '/admin'. 
```
const appRoutes: Routes = [
  { path: 'admin', component: AdminComponent },
  { path: '',
    redirectTo: '/admin',
    pathMatch: 'full'
  },
  { path: '**', component: PageNotFoundComponent },
....
];
```
make sense?

**Comment by MarkRobertJohnson** [comment]
Yes, not really an option as I do not want the path '' to redirect to the 'admin'.  My preference would be to allow a custom redirect destination path after the login completes.  

I found that instead of just doing a "this.router.navigate([''])" in the ezcode-adal-callback.guard.ts, allowing for a custom redirect path in the "this.router.navigate(['/admin'])"  accomplished exactly what I need, but I found that the Config on the ADAL Service was not available at this point.

At this point I have to create a fork and hardcode my path into that callback component
